setup.cfg

```diff
@@ -12,12 +12,11 @@ tag_prefix =
 parentdir_prefix = modin-
 
 [tool:pytest]
-addopts = --cov-config=setup.cfg --cov=modin --cov-append --cov-report=
+addopts = --cov-config=setup.cfg --cov=modin --cov-append --cov-report= -m "not exclude_by_default"
 xfail_strict=true
 markers =
-    xfail_executions
-    skip_executions
     exclude_in_sanity
+    exclude_by_default
```

Comment:
is this a built-in pytest marker? I couldn't find info about it anywhere https://docs.pytest.org/en/stable/search.html?q=exclude_by_default

Reply:
No, these markers are custom, they can be defined by the user himself, which is what I did.

Reply:
then I don't quite understand how do we use it. Is its value `True` by default?

Reply:
By default - `False`.

For example:
```bash
pytest modin\pandas\test\dataframe\test_indexing.py::test_reset_index_with_multi_index_drop --co -m exclude_by_default
====== 944/1040 tests collected (96 deselected) in 3.12s ======

vs

pytest modin\pandas\test\dataframe\test_indexing.py::test_reset_index_with_multi_index_drop --co
====== 96/1040 tests collected (944 deselected) in 4.30s ======
```

Reply:
Oh, there are three cases...

I expected this to work something like:
```
pytest modin/ # run everything
pytest modin/ -m exclude_by_default # skip tests that are marked as 'exclude_by_default'
```

Maybe we should rename the variable and stick to the following behaviour?
```
pytest modin/ # run limited testing
pytest modin/ -m long_testing # run all the tests 
```

Reply:
> pytest modin/ -m long_testing # run all the tests 

@dchigarev I can change the name to `long_testing`, but only tests with that mark will run.

To run all tests we still have to use `-m "long_testing or not long_testing"`

Reply:
> To run all tests we still have to use -m "long_testing or not long_testing"

Do I understand correctly, that the new approach with a custom pytest mark, doesn't allow us to simply turn on/off extra tests? Why do we need to change the existing approach then?

Reply:
> Do I understand correctly, that the new approach with a custom pytest mark, doesn't allow us to simply turn on/off extra tests? Why do we need to change the existing approach then?

Because it is still simple, and at the same time it is a **standard** way for `pytest` to disable additional test cases.